lib/widgets/zoomable_optimized_directus_image.dart

```diff
@@ -0,0 +1,54 @@
+import "package:flutter/material.dart";
+import "../theme/colors.dart";
+import "optimized_directus_image.dart";
+
+class ZoomableOptimizedDirectusImage extends StatelessWidget {
+  const ZoomableOptimizedDirectusImage(
+    this.imageUrl, {
+    super.key,
+    this.noShimmeringLoading = false,
+    this.boxFit = BoxFit.cover,
+  });
+
+  final String? imageUrl;
+  final bool noShimmeringLoading;
+  final BoxFit boxFit;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () async {
+        await _showFullScreenImage(context);
+      },
+      child: OptimizedDirectusImage(
+        imageUrl,
+        boxFit: boxFit,
+        noShimmeringLoading: noShimmeringLoading,
+      ),
+    );
+  }
+
+  Future<void> _showFullScreenImage(BuildContext context) async {
+    await Navigator.of(context).push(
+      MaterialPageRoute(
+        builder: (newContext) => Scaffold(
```

Comment:
using `newContext` instead of just `context` can lead to mistakes

Reply:
the convention is just to use `context` always

Reply:
Changed